tools/webcam/configs/matchstickmen/README.md

```diff
@@ -0,0 +1,35 @@
+# MatchStickMen and ElkHorn Effects
+
+We can apply MatchStickMen and ElkHorn effects on videos with pose estimation results.
```

Comment:
The description is a little too long. We don't need to tell users the details like "modifying skeleton links". Just describe how to play with the demo, like:
- be prepared for the matchstick version of yourself with an elkhorn.
- What's more? Try gesturing a heart with your hands and see what will happen :)

Reply:
Fixed.